File: web/WebFirstLab/php/TextFieldContainer.php
<?php
    require_once('Containable.php');
    require_once('WrongInputException.php');

    class TextFieldContainer implements Containable {
        private $containable;
        private $lowerBound;
        private $upperBound;

        public function __construct(Containable $containable, $lowerBound, $upperBound) {
            $this->containable = $containable;
            $this->lowerBound = $lowerBound;
            $this->upperBound = $upperBound;
        }

        public function content() {
            $content = $this->containable->content();

            if($this->lowerBound <= $content && $content <= $this->upperBound) {
                return $content;
            }

            throw new WrongInputException();
        }
    }
?>

File: web/WebSecondLab/src/main/java/app/entities/classes/TextFieldContainer.java
package app.entities.classes;

import app.entities.exceptions.WrongInputException;
import app.entities.interfaces.Containable;

public class TextFieldContainer implements Containable<Float> {
    private final Containable<Float> containable;
    private final float lowerBound;
    private final float upperBound;

    public TextFieldContainer(Containable<Float> containable, float lowerBound, float upperBound) {
        this.containable = containable;
        this.lowerBound = lowerBound;
        this.upperBound = upperBound;
    }

    @Override
    public Float content() throws WrongInputException {
        float content = containable.content();

        if (content >= lowerBound && content <= upperBound) {
            return content;
        }

        throw new WrongInputException("Out of correct range");
    }
}
